Pipfile.lock

```diff
@@ -388,7 +411,7 @@
                 "sha256:cf166cfc5b8197090ba1d90fddf34e37e9216c0452b4700553b3c09b0629ff2f"
             ],
             "index": "pypi",
-            "version": "==0.13.4"
```

Comment:
The aicoe-ci/build-check failed as the sha was not updated.

Reply:
We should close this PR, has it would be a wrong piplock

Reply:
Yes, let's merge it so a new one can be opened with newly built kebechet container image :+1: